cmd/finch/nerdctl_darwin_test.go

```diff
@@ -1171,6 +1171,37 @@ func TestNerdctlCommand_run(t *testing.T) {
 				c.EXPECT().Run()
 			},
 		},
+		{
```

Comment:
we should include all long form flags supported by nerdctl in the check, irrespective of the ones present in our dictionary.

Reply:
```
--debug                   
--debug-full
--experimental
--insecure-registry
--attach
--detach
--interactive
--oom-kill-disable
--privileged
--read-only
--init
--rm
--sig-proxy
--tty

``` 
These are the long form boolean commands available on nerdctl. (` nerdctl run --help | grep -- '--' | grep -v -E 'string|int|float|uint|uint16' | grep -v '=' `)

I have included everything except
```
--debug - since debug-full is already present
--insecure-registry - this is a pull specific flag
--attach - contradicts detach
```

It is important to note that certain global commands ``--experimental`` and ``--debug-full`` result in the boolean value being sent as argument instead of an equality sign (e.g ``--debug-full false``), it is valid nerdctl command though. Just something to note.